src/sass/main.css

```diff
@@ -27,4 +27,73 @@
   background-image: url("../images/demo.jpg");
   background-size: cover;
   background-position: center;
+}
+
+.label {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  border: 1px solid #d7d7d7;
+  border-radius: 4px;
+}
+.label__container {
+  display: flex;
+  align-items: center;
+  flex-direction: column-reverse;
```

Comment:
to stylowanie wynika z wielu prób różnych stylowań. Próbowałem bawić się w stylowanie potomków itp. Możliwe że gdzię się zapędziłem i nie będzie to praktyczne